### DISCUSSION TITLE
Allow following the best practice of not using library()

### DISCUSSION THREAD

**Discussion by thk686** [ropensci/targets#1193] [Help]
I was just setting up a targets workflow and tried to use the best practice of using requireNamespace and the pkg::function calling convention throughout my code. This fails as targets cannot figure out the workflow and gives a "not a dag" error. My proposed feature is that you not depend on calling the library function.

**Comment by wlandau** [comment]
@thk686, do you have a reprex? Not sure I see the connection among the issues you cited above. Please have a look at https://books.ropensci.org/targets/help.html#write-a-reprex.
